mailexporter.conf

```diff
@@ -3,9 +3,6 @@
 # time between two monitoring-attempts
 monitoringinterval: 10m
 
```

Comment:
Please delete one of the empty lines surrounding the old option, so that the remaining options are separated by one single empty line.